### PR TITLE
docs: add documentation for xapi concepts

### PR DESCRIPTION
**Pull request by Ian2012**
## Description

This PR adds documentation for xAPI concepts

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @Ian2012!  Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

File: docs/concepts/xapi_concepts.rst [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+Learning Record Store (LRS)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The Learning Record Store serves as the repository for storing and retrieving xAPI
+statements. It is a database or storage system that receives and securely stores
+the statements generated by learning activities. The LRS enables the collection 
+and organization of learning data from various sources, allowing for robust 
+analysis and reporting of learner experiences.
+
+Together, these components form the foundation of the xAPI ecosystem, enabling the tracking and analysis of learner activities and experiences beyond traditional learning management systems. The flexibility and interoper
```
[the remark this answers is not in the document]
Comment (Ian2012) [reply docs/concepts/xapi_concepts.rst:43]: I was waiting for a confirmation on where to put the warning, what do you think?

[the remark this answers is not in the document]
Comment (bmtcril) [reply docs/concepts/xapi_concepts.rst:43]: Ahh, I think this doc explains what an actor is we can point to it from another doc. Probably a "How To Change the Actor Identifier" that links back to the actor concept, and describes what the various setting values do. That's where I'd put the warning.

**Comment by Ian2012** [comment]
@bmtcril ready for review

**Review by bmtcril — COMMENTED** [review]
Looks good overall, just a few comments

File: docs/how-tos/changing_actor_identifier.rst [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+identifier is the anonymous user ID which is different per user per course. 
+If you want to change this behavior, you can update the value of `XAPI_AGENT_IFI_TYPE` 
+to change the identifier. The supported values are:
+
+* `mbox`:  the email is used
+* `mbox_sha1sum`: the sha1sum of the email is used
+
+In the case, the `XAPI_AGENT_IFI_TYPE` is different than those values the anonymous user ID is used.
+
+Changing the xAPI actor identifier will invalidate all the previous statements in the data-lake
```
[the remark this answers is not in the document]
Comment (bmtcril) [reply docs/how-tos/changing_actor_identifier.rst:18]: I think this one still needs to be addressed

**Review by bmtcril — APPROVED** [review]

**Comment by openedx-webhooks** [comment]
@Ian2012 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/openedx-oars&entry.867055334=https://github.com/openedx/openedx-oars/pull/54&entry.1484655318=https://github.com/Ian2012&entry.752974735=2023-06-08+20:19&entry.1917517419=2023-06-15+17:29&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**